doc/Tests.md

```diff
@@ -0,0 +1,18 @@
+# Themes
```

Comment:
Can you link that document from the `Home.md` and `de/Home.md` file so that it is visible from the `/help` pages on the nodes.

Reply:
Should I add it to `de` even if the page is not translated yet?

Reply:
Yes, the documents in English have a `(EN)` at the end. It's better to have the docs accessible even if they are in English then to hide them when the DE doc index is loaded.

Reply:
I added it (7893d96ad8847455bcb0f9672e7d5381d698f29d).